addon/utils/model-factory.js

```diff
@@ -71,21 +87,19 @@ export function getTopLevelPropertyGroups({ store, graphs, form }) {
   if (
     getFormModelVersion(form, {
       store,
-      formGraph: graphs.formGraph,
+      formGraph: formGraph,
     }).toLowerCase() == 'v2'
```

Comment:
nitpick: Should we make a constantout of the version or is this a one time thing to use here?
Thinking `v2` or `V2` or `version2`